package.json

```diff
@@ -128,6 +128,228 @@
                 "language": "csharp",
                 "path": "./snippets/csharp.json"
             }
+        ],
+        "debuggers": [ 
+            {
+                "type": "coreclr",
+                "label": ".NET Core",
+                "enableBreakpointsFor": { "languageIds": [ "csharp" ] },
+
+                "program": "./coreclr-debug/debugAdapters/OpenDebugAD7",
+                "windows": {
+                    "program": "./coreclr-debug/debugAdapters/OpenDebugAD7.exe"
+                },
+
+                "configurationAttributes": {
+                "launch": {
+                    "required": [ "program", "cwd" ],
+                    "properties": {
+                    "program": {
+                        "type": "string",
+                        "description": "Path to the program (executable file) to launch. On Windows, a '.exe' suffix is appended if not specified already.",
+                        "default": "${workspaceRoot}/bin/Debug/dnxcore50/<My-Project-Name>"
```

Comment:
We should merge in the most recent changes that @caslan made to this (removal of 'dnxcore50').